src/main/kotlin/com/hibob/academy/resource/OwnerApi.kt

```diff
@@ -11,13 +13,14 @@ import java.util.concurrent.CopyOnWriteArrayList
 @Path("/api/adi/owners")
 @Produces(MediaType.APPLICATION_JSON)
 @Consumes(MediaType.APPLICATION_JSON)
-class OwnerResource {
+class OwnerResource(private val ownerService: OwnerService) {
 
     private val allOwners: MutableList<Owner> = CopyOnWriteArrayList()
+    private val companyId = 2L
 
     @GET
     @Path("/{ownerId}")
-    fun getOwnerById(@PathParam("ownerId") ownerId: Long): Response {
+    fun getOwnerById(@PathParam("ownerId") ownerId: Long?): Response {
```

Comment:
OwnerId can be nullable?
If we want to get the owner by id then shouldn't be nullable